requirements/base.txt

```diff
@@ -1,11 +1,12 @@
-django==1.9.6                            	# Our web framework
+django==1.9.6                               # Our web framework
 django-countries==3.4.1
 dj-database-url==0.4.1
-gunicorn==19.5.0							# Python WSGI HTTP Server
+gunicorn==19.5.0                            # Python WSGI HTTP Server
 psycopg2==2.6.1
 whitenoise==3.1
-django-custom-user==0.6.0					# Custom user model to allow user auth via email
+django-custom-user==0.6.0                   # Custom user model to allow user auth via email
 wsgiref==0.1.2
-django-s3direct==0.4.2						# Handles asset uploads to s3
-django-registration-redux==1.4				# Double opt-in user registration flow
-python-env-utils==0.2.0						# Deals nicely with environment var typing
+django-s3direct==0.4.2                      # Handles asset uploads to s3
+django-registration-redux==1.4              # Double opt-in user registration flow
+python-env-utils==0.2.0                     # Deals nicely with environment var typing
+jango-widget-tweaks==1.4.1                  # Help render and tweak forms
```

Comment:
The details page looks very nice! just something I noticed. My first name is lower case.. maybe we could capitalise the first letter or the First/Last names in case users are typing lower case?